lib/membrane/rtp/jitter_buffer.ex

```diff
@@ -181,16 +191,13 @@ defmodule Membrane.RTP.JitterBuffer do
   defp record_to_action(nil), do: {:event, {:output, %Membrane.Event.Discontinuity{}}}
   defp record_to_action(%Record{buffer: buffer}), do: {:buffer, {:output, buffer}}
 
-  @spec get_and_update_stats(State.t()) :: {Stats.t(), State.t()}
-  def get_and_update_stats(%State{
-        store: store,
-        stats: %{expected_prior: expected_prior, received_prior: received_prior, jitter: jitter}
-      }) do
-    use Bitwise
+  @spec get_updated_stats(State.t()) :: {Stats.t(), State.t()}
+  defp get_updated_stats(state) do
+    %State{store: store, stats_acc: stats_acc} = state
 
     # Variable names follow algorithm A.3 from RFC3550 (https://tools.ietf.org/html/rfc3550#appendix-A.3)
-    %{base_index: base_seq, end_index: extended_max, received: received} = store
-    expected = extended_max - base_seq + 1
+    %BufferStore{base_index: base_seq, end_index: extended_max, received: received} = store
+    expected = if base_seq == nil, do: 0, else: extended_max - base_seq + 1
```

Comment:
This seems like trouble - what kind of stats we can generate if nothing was received (only then base_seq is `nil`)? Instead of doing this if, I'd rather prevent calling this function in such state

Reply:
> what kind of stats we can generate if nothing was received

Empty stats. We have to reply somehow for the stats request, why not reply with empty stats?

Reply:
Struct filled with 0s is not a good indicator of "not ready to send stats, don't include it in RTCP packet"